plugin/queries.js

```diff
@@ -130,21 +130,6 @@ const productsQuery = (date) => `
                   }
                 }
               }
-              presentmentPrices {
```

Comment:
These don't come back with an id so it's not clear how to go about sourcing them correctly. If the solution to this type of problem is known and simple, I'd be glad to implement it. Otherwise, we should find out whether our customers use this data before sinking too much energy into it.